### DISCUSSION TITLE
Who is using NativeScript in Production?

### DISCUSSION THREAD

**Discussion by NathanWalker** [NativeScript/NativeScript#9473] [Show and tell]
If you are using NativeScript for production deployments in any shape or form (Angular, React, Svelte, Vue, Ionic, Capacitor), feel free to share!

You can [view the NativeScript showcase here as well](https://nativescript.org/showcase/)

**Comment by farfromrefug** [comment]
* OSS Weather (Svelte) 
    - Android: https://play.google.com/store/apps/details?id=com.akylas.weather&hl=en_US&gl=US   
    - IOS: https://apps.apple.com/us/app/ns-weather/id1499117252
 * Elsi Station (Vue) 
    - Android: https://play.google.com/store/apps/details?id=com.elichens.stationmobile   
    - IOS: https://apps.apple.com/us/app/elsi-station/id1538561086
 * eLichens Air (Vue) 
    - Android: https://play.google.com/store/apps/details?id=com.elichens.outdoorapp
    - IOS: https://apps.apple.com/us/app/elichens-air/id1552617144
 * Aura CO2 (Vue) 
    - Android: https://play.google.com/store/apps/details?id=com.elichens.auraco2
    - IOS: https://apps.apple.com/us/app/aura-co2/id1565190899
* ActiveLook (Vue):
    - Android: https://play.google.com/store/apps/details?id=com.microoled.activelook.mobile&hl=fr&gl=US
    - iOS: https://apps.apple.com/app/activelook-sport-glasses/id1464023773
    
   More to come soon...

**Comment by shiv19** [comment]
Grainfather Community
An app for Home Brewers
Android: http://bit.ly/grainfatherandroid
iOS: http://bit.ly/grainfatherios

**Comment by davecoffin** [comment]
Daily Nanny:
iOS: https://apps.apple.com/us/app/daily-nanny/id1180728044
Android: https://play.google.com/store/apps/details?id=com.cubbynotes.pocketnanny&hl=en_US&gl=US

MyPreSonus:
iOS: https://apps.apple.com/us/app/mypresonus/id1282534772
Android: https://play.google.com/store/apps/details?id=com.presonus.mypresonus&hl=en_US&gl=US

FedGovSpend:
iOS: https://apps.apple.com/us/app/fedgovspend-explorer/id1543742755
Android: https://play.google.com/store/apps/details?id=edu.unh.fedgovspend.explorer

I can assemble more later :)

**Comment by vallemar** [comment]
Checkpoint Curly
iOS: https://apps.apple.com/us/app/checkpoint-curly/id1549378423#?platform=iphone
Android: https://play.google.com/store/apps/details?id=com.vallemar.checkpointcurly

**Comment by felixkrautschuk** [comment]
FanFiktion.de
Android: https://play.google.com/store/apps/details?id=net.fanfiktion.mobile
iOS: https://apps.apple.com/de/app/fanfiktion-de/id820349064

Dresden App
Android: https://play.google.com/store/apps/details?id=dig.dresden.apps
iOS: https://apps.apple.com/de/app/offizielle-dresden-app/id1065000438

**Comment by cjohn001** [comment]
My-Nutri-Diary 
The premium nutrition diary. 100% free of charge. 
Learn to understand your metabolism and achieve your fitness and health goals.

iOS: https://apps.apple.com/de/app/my-nutri-diary/id1490340361#?platform=iphone
Android coming soon: https://play.google.com/store/apps/details?id=de.mynutridiary.mnd&hl=en_US&gl=US

**Comment by madmas** [comment]
**Debeka Mitarbeiter Foto-App** (https://www.youtube.com/watch?v=R-lNcfrMFSM)
(only useable for Debeka agents)
iOS (internal only / enterprise distribution certificate)
Android https://play.google.com/store/apps/details?id=de.debeka.fotoapp

**Debeka Auslands-SOS**
iOS: https://apps.apple.com/de/app/debeka-auslands-sos/id537548994
Android: https://play.google.com/store/apps/details?id=de.debeka.mobile

**Comment by EdJones** [comment]
**Science of Reading** 
Half of US schools of education still teach a flawed model of how kids learn to read; your kids' school probably uses it.
iOS: https://apps.apple.com/us/app/science-of-reading/id1480512339
Android: coming.

**Comment by kakha13** [comment]
**Never Have I Ever Online**
[Android](https://play.google.com/store/apps/details?id=ge.any.NeverHaveIEver)
[Web](https://neverhaveiever.online/)

**Georgian Air Quality**
[Android ](https://play.google.com/store/apps/details?id=ge.any.georgianairquality)
[iOS](https://apps.apple.com/ge/app/georgian-air-quality/id1554751727)

**Georgian Megrelian English Dictionary**
[Android](https://play.google.com/store/apps/details?id=ge.any.qartulmegrulileqsikoni)
[iOS](https://apps.apple.com/ge/app/georgian-air-quality/id1554751727)

**Medical Dictionary**
[Android](https://play.google.com/store/apps/details?id=ge.any.MedicalDictionary)

**Georgian Svan English Dictionary**
[Android](https://play.google.com/store/apps/details?id=ge.any.svanurileqsikoni)

**Comment by NathanWalker** [comment]
- **Sweet - Digital Collectibles** (Angular)
  - iOS: https://apps.apple.com/us/app/sweet-digital-collectibles/id1452120535

**Comment by Bezlepkin** [comment]
**Mall’ER Play**

iOS: https://apps.apple.com/ru/app/maller-play/id1577740837
Android: https://play.google.com/store/apps/details?id=com.maller.starnumber

Perhaps the complexity has gone beyond the limits of a cross-platform application :) 5 types of cameras, a bunch of plugins :)

**Comment by AnthonyLenglet** [comment]
**Edit: this app has since been redone in react native, due to poor community support for the framework along with bad performance leading to negative reviews**

ÏDTROC:

- iOS: https://apps.apple.com/us/app/idtroc/id1484937026
- Android: https://play.google.com/store/apps/details?id=com.idtroc.idkids

Used for clothing sale in over 300 stores across France/Belgium/Switzerland (with the goal of propagating toward the rest of Europe)

**Reply by rigor789** [reply]
Hey @AnthonyLenglet that looks awesome, would you be interested in submitting it to the showcase? 
https://github.com/NativeScript/showcase

**Comment by kefahB** [comment]
LightPAY is a digital meal voucher payment application 👍 
[IOS](https://apps.apple.com/app/apple-store/id1463962129)
[Android](https://play.google.com/store/apps/details?id=com.clients.digipay)

**Comment by PhilippS93** [comment]
enduco is a AI based coaching app for sportmen and sportswomen
https://apps.apple.com/de/app/enduco-k-i-personal-trainer/id1483216578
https://play.google.com/store/apps/details?id=app.enduco.mobile&hl=de&gl=US
